lwc/src/main/scala/quasar/physical/parsing/package.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2014–2018 SlamData Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// from circe/circe-fs2 0.9.1
+
+package quasar.physical.s3
+
+import slamdata.Predef.{Seq => _, _}
+import scala.collection.Seq
+
+import _root_.fs2.{ Chunk, Pipe, Stream }
+import _root_.fs2.pipe
+import _root_.jawn.{ AsyncParser, ParseException }
```

Comment:
I find imports a lot less readable with `_root_` prefixing them, so as a style concern, I'd prefer if we didn't use `_root_` unless we are actually shadowing package names.

Reply:
This was just a holdover from copy/paste, the package was `circe.fs2`. I'll fix that.